resources/conf/config.toml

```diff
@@ -106,6 +120,19 @@
   publicCertificatePath = "/home/wso2/security/truststore/mg.pem"
   privateKeyPath = "/home/wso2/security/keystore/mg.key"
 
+[enforcer.jwtIssuer]
+  enabled = true
+  issuer = "https://localhost:9095/token"
+  encoding = "base64" # base64,base64url
+  claimDialect = ""
+  signingAlgorithm = "SHA256withRSA"
+  publicCertificatePath = "/home/wso2/security/truststore/mg.pem"
+  privateKeyPath = "/home/wso2/security/keystore/mg.key"
+  validityPeriod = 3600 # seconds
+  [[enforcer.jwtIssuer.jwtUser]]
+    username = "admin"
+    password = "admin"
```

Comment:
Need to replace the password with "$env{enforcer_admin_pwd}"

Didn't add the change as the value didn't get resolved while testing.